frontend/src/contexts/HostContext/HostReducer.js

```diff
@@ -373,6 +373,16 @@ function toggleJoinCode(state) {
   };
 }
 
+function removeLastPlayerFromStaging(state) {
+  return {
+    ...state,
+    newPlayerStaging: state.newPlayerStaging.slice(
+      0,
+      state.newPlayerStaging.length - 1,
+    ),
+  };
+}
```

Comment:
Since the rest of this functionality is setup to reject multiple people at the same time I would like to see this match all of that other functionality rather than just removing the last player from  `newPlayerStaging`.